src/Plugin/WebformElement/CivicrmSelect.php

```diff
@@ -77,7 +77,9 @@ public function prepare(array &$element, WebformSubmissionInterface $webform_sub
       }
       $element['#options'] = $new;
     }
-    $element['#default_value'] = $element['#default_option'];
+    if (isset($element['#default_option'])) {
+      $element['#default_value'] = $element['#default_option'];
+    }
   }
```

Comment:
Ran into this `Notice: Undefined index: #default_option in Drupal\webform_civicrm\Plugin\WebformElement\CivicrmSelect->prepare() (line 80 of /var/www/drupal/web/sites/default/modules/webform_civicrm/src/Plugin/WebformElement/CivicrmSelect.php)`